webcomponents/input/input.webcomponent.spec.js

```diff
@@ -183,6 +183,16 @@ describe('mn-input (webcomponent)', () => {
   })
 
   describe('attribute readonly', () => {
+    it('should define attribute inside of a element', () => {
```

Comment:
or "should add readonly to input child"

or "should add attribute to input child"